internal/command/versions.go

```diff
@@ -0,0 +1,48 @@
+package command
+
+import (
+	"fmt"
+	"io/ioutil"
+	"os"
+	"os/exec"
+	"path"
+
+	"github.com/cyberark/summon/pkg/summon"
+	"github.com/cyberark/summon/provider"
+)
+//Create the AllVersions string to include versions of all the providers
+const SUMMON_VERSION = summon.VERSION
+var DefaultPath = provider.DefaultPath
+var AllVersions = GetVersions(GetProviderFileNames())
+
+//Run `$provider --version`(from DefaultPath),
+// 	for each provider in the list of providers
```

Comment:
These two comments have a really short line length. Stick to 80-100 chars for line breaks in comments/code